### ISSUE TITLE
CheckPoint option

### ISSUE THREAD

**Issue by QuandoSJ** [kentaroy47/frcnn-from-scratch-with-keras#70]
Is there a option for me to save checkpoint while training and then continue training from the checkpoint. Cause right now I cannot start from where I stopped but to start over again

**Comment by magloiretouh** [comment]
The model is saved at the end of each epoch if the total loss is less than the previous epoch total loss. This is done using a [ModelCheckpoint callback](https://keras.io/callbacks/#modelcheckpoint). So to resume where you left off, you just have to use the load option the next time you want to initialize your training
